content/ru/docs/concepts/workloads/pods/_index.md

```diff
@@ -0,0 +1,395 @@
+  kind: "Под"
+content_type: concept
+weight: 10
+no_list: true
+---
+
+<!-- overview -->
+
+_Поды_, _Pods_ - это самые маленькие развертываемые вычислительные единицы, которые можно создавать и которыми 
+можно управлять в Kubernetes.
+
+_Под_ (как стая китов или гороховый стручок) - это группа из одного или нескольких 
+{{< glossary_tooltip text="контейнеров" term_id="container" >}} с общими ресурсами хранения и сети, 
+а также спецификация для запуска контейнеров.
+Содержимое Пода всегда находится в одном месте, имеет совместное расписание и выполняется в общем 
+контексте. Под моделирует специфический для приложения "логический хост": он содержит один или несколько 
+контейнеров приложений, которые относительно тесно связаны между собой.
+В необлачных контекстах приложения, выполняемые на одной физической или виртуальной машине, 
+аналогичны облачным приложениям, выполняемым на одном логическом хосте.
+
+Помимо контейнеров приложений Под может содержать 
+{{< glossary_tooltip text="init контейнеры" term_id="init-container" >}}, которые запускаются при 
+старте Пода. Вы также можете внедрить 
+{{< glossary_tooltip text="эфемерные контейнеры" term_id="ephemeral-container" >}} 
+для отладки запущенного Пода.
+
+<!-- body -->
+
+## Что такое Под? {#what-is-a-pod}
+
+{{< note >}}
+Вам нужно установить [исполняемую среду контейнеров](/docs/setup/production-environment/container-runtimes/) 
+на каждый узел кластера, чтобы на нем могли работать подсистемы.
+{{< /note >}}
+
+Общий контекст Пода - это набор пространств имен Linux, cgroups и, возможно, других аспектов изоляции - 
+того же, что изолирует {{< glossary_tooltip text="контейнер" term_id="container" >}}. Внутри контекста Пода 
+отдельные приложения могут иметь дополнительные субизоляции.
+
+Под похож на набор контейнеров с общими пространствами имен и общими томами файловой системы.
+
+Поды в кластере Kubernetes используются двумя основными способами:
+
+* **Поды, которые запускают один контейнер**. Модель "один контейнер на Под" является наиболее 
+  распространенным вариантом использования Kubernetes; в этом случае вы можете рассматривать Под как 
+  обертку вокруг одного контейнера; Kubernetes управляет Подами, а не непосредственно контейнерами.
+* **Поды, которые запускают несколько контейнеров, обязанных работать вместе**. Под может инкапсулировать 
+  приложение, состоящее из [нескольких расположенных рядом контейнеров](#how-pods-manage-multiple-containers), 
+  которые тесно связаны между собой и нуждаются в совместном использовании ресурсов. Эти расположенные 
+  рядом контейнеры образуют единое целое.
+  
+  Группировка нескольких совместно расположенных и совместно управляемых контейнеров в одном Поде - это 
+  довольно сложный случай использования. Этот паттерн следует использовать только в особых случаях, когда 
+  ваши контейнеры тесно связаны между собой.
+
+  Вам не нужно запускать несколько контейнеров для обеспечения репликации (для обеспечения устойчивости 
```

Comment:
We try to avoid repeating "you" in Russian, which is normal for English.
```suggestion
  Запускать несколько контейнеров для обеспечения репликации (устойчивости 
```